src/lib/select/select.ts

```diff
@@ -313,13 +318,18 @@ export class MdSelect implements AfterContentInit, OnDestroy, OnInit, ControlVal
   constructor(private _element: ElementRef, private _renderer: Renderer2,
               private _viewportRuler: ViewportRuler, private _changeDetectorRef: ChangeDetectorRef,
               @Optional() private _dir: Dir, @Self() @Optional() public _control: NgControl,
-              @Attribute('tabindex') tabIndex: string) {
+              @Attribute('tabindex') tabIndex: string,
+              @Optional() @Inject(MD_PLACEHOLDER_GLOBAL_OPTIONS)
+              placeholderGlobalOptions: PlaceholderOptions) {
```

Comment:
same, change the var name and format like input constructor so it can fit on one line